package.json

```diff
@@ -8,18 +8,18 @@
   },
   "main": "lib/index.js",
   "scripts": {
-    "test": "npm run lint && npm run test-functional",
-    "lint": "jscs `find ./lib -name '*.js'` -v && jscs `find ./test -name '*.js'` -v",
-    "test-functional": "tap `find ./test/functional -name '*.test.js'`",
-    "test-system": "tap `find ./test/system -name '*.test.js'`",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "lint": "npm run eslint",
+    "eslint": "./node_modules/.bin/eslint -c .eslintrc lib test",
+    "test": "npm run lint && npm run test-functional && npm run test-system",
+    "test-functional": "tap ./test/functional/*.test.js -R spec",
+    "test-system": "tap ./test/system/*.test.js -R spec --timeout=60",
+    "semantic-release": "semantic-release"
   },
   "author": "snyk.io",
   "license": "Apache-2.0",
   "devDependencies": {
-    "jscs": "^3.0.7",
-    "semantic-release": "^6.3.6",
-    "tap": "^10.3.2",
-    "tap-only": "0.0.5"
+    "eslint": "^5.3.0",
+    "semantic-release": "^15",
+    "tap": "^12.0.1"
```

Comment:
I love `tap-only` - would prefer to leave it. It's useful when debugging - can easily make a single test to run without commenting out code